src/generate.ts

```diff
@@ -22,5 +28,101 @@ export function generate(specQ: SpecQuery, schema: Schema, opt: QueryConfig = DE
     }
   });
 
+  let ENCODING_QUERY_INDEX: Dict<EncodingQuery> = {};
```

Comment:
Because of that -- create a new file called `stylize.ts`  that contains a main `stylize` method and multiple stylizer in it that get triggers based on config (I think it's better than `modifier.ts`).